javascript/javascript1/week2/flightBooking.js

```diff
@@ -0,0 +1,18 @@
+function getFullname(firstname, surname, useFormalName = false, title = "Mx.") {
```

Comment:
Mx is gender-neutral, challenge yourself to check if the person you add is female or not :)